src/surface/classes.scss

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './index';
+
+.iui-surface {
+  &,
+  &-2 {
```

Comment:
I would say that these should be modifier classes like `iui-surface iui-surface-elevation-2`

Reply:
What's the benefit of modifiers?

Reply:
Having a surface without a shadow didn't seem useful, so I made the 2 base classes equal the smallest shadow.  I could make the class names go by 1, it's matching the box-shadow variables now.

Reply:
Should we use CSS variables to allow easily customizing elevation, background and border-radius from the React component?

```tsx
const Surface = ({ elevation = 1, background = 1, borderRadius = 1, ...props }) => {
  return (
    <div
     {...props}
     style={{
       'iui-surface-elevation': `var(--iui-elevation-${elevation})`,
       'iui-surface-background': `var(--iui-color-background-${background})`,
       'iui-surface-border-radius': `var(--iui-border-radius-${borderRadius})`,
     }}
    />
  );
};
```

Usage:
```tsx
<Surface elevation={2} background={2} borderRadius={3}>...</Surface>
```

Reply:
A couple of changes:
- You can apply the surface using classes `.iui-surface .iui-elevation-1`.
- You can apply the surface using mixin `@include iui-surface;`.
- Elevations are now `.iui-elevation-1` - `.iui-elevation-5` & no number defaults to `.iui-elevation-1`.

Reply:
I think it makes sense for our menu component to use the mixin because the menu should always have that surrounding box with drop shadow.
However, for date picker it might make more sense to use the classes because you may want to embed the calendar somewhere in a page w/o the drop shadow.

Reply:
> Should we use CSS variables to allow easily customizing elevation, background and border-radius from the React component?

Then we would also convert SASS variables for elevation and others to CSS variables?

Reply:
Latest push switches to CSS variables, let me know if this is not what you were expecting.

Reply:
yes, we don't need those extra classes either because we have CSS variables

Reply:
How should apply the various elevations on the surface demo page without the modifier classes?

Reply:
Use inline CSS variables with hardcoded values.

We should in the future move those Sass variables to CSS variables so that we can reference them in the inline styles.